extensionsGallery.json

```diff
@@ -3603,14 +3603,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.2.0",
-							"lastUpdated": "10/14/2020",
+							"version": "0.3.2",
```

Comment:
Are you purposely shipping this out of band? Or is this intended to go out in the stable release? If the latter you should follow the process documented here : https://github.com/microsoft/azuredatastudio-docs/blob/main/Guides/UpdateExtensionGallery.md

and update this branch instead of making a separate one for just the Kusto extension : https://github.com/microsoft/azuredatastudio/tree/release/extensions-rc

Reply:
Sure, will create a separate PR on extension-rc branch